### ISSUE TITLE
Exception on sparse Arrays

### ISSUE THREAD

**Issue by arikw** [j-brooke/FracturedJsonJs#13]
When running the method on a sparse array (e.g., ['val1', , , 'val2']), an exception occurs. 

See the following example:
```js
import { Formatter, FracturedJsonOptions, EolStyle, NumberListAlignment } from 'fracturedjsonjs';
function stringify(object) {
  const options = new FracturedJsonOptions();
  options.MaxTotalLineLength = 100;
  options.AlwaysExpandDepth = 2;
  options.IndentSpaces = 2;
  options.JsonEolStyle = EolStyle.Lf;
  options.OmitTrailingWhitespace = true;
  options.normalizeNumberPrecision = NumberListAlignment.Left;

  const formatter = new Formatter();
  formatter.Options = options;

  return formatter.Serialize(object);
}

stringify(['val1', , , 'val2']); // exception
```

**Comment by j-brooke** [comment]
Thanks for reporting this.  I'll have a fix out soon.

**Comment by j-brooke** [comment]
Fixed in version 4.0.2.
